basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,129 @@
-const helpers = require('./project-1');
+const helpers = require("./project-1");
 
 // start testing!
+describe("helpers.js", () => {
+  // multiplyByTen
+  test("tests 10 multiply 10 equals 100", () => {
+    expect(helpers.multiplyByTen(10)).toBe(100);
+  });
+
+  // subtractFive
+  test("tests if 10 subtract returns 5", () => {
+    expect(helpers.subtractFive(10)).toBe(5);
+  });
+
+  // areSameLength
+  test("tests if cat and dog are same length and returns true", () => {
+    expect(helpers.areSameLength("cat", "dog")).toBe(true);
+  });
+
+  // areEqual
+  test("tests if 5 and 5 are equal and returns true", () => {
+    expect(helpers.areSameLength(5, 5)).toBe(true);
+  });
+
+  // lessThanNinety
+  test("tests if 40 is less than ninety and returns true", () => {
+    expect(helpers.lessThanNinety(40)).toBe(true);
```

Comment:
There's actually a method called toBeTruthy() you can use instead of toBe(true).